roxie/ccd/ccdcontext.cpp

```diff
@@ -3898,6 +3902,30 @@ class CRoxieServerContext : public CRoxieContextBase, implements IRoxieServerCon
         clusterNames.pop();
         clusterWidth = -1;
     }
+
+    virtual bool logStartStopError()
+    {
+        if (actResetLogState == LogResetInit)
+        {
+            unsigned timeNow = msTick();
+            unsigned timePrev = serverQueryFactory->getTimeActResetLastLogged();
```

Comment:
Not sure if we wanted a time since last logged for each query or a global time ?